internal/sdk/resource_decode.go

```diff
@@ -57,6 +58,7 @@ func decodeReflectedType(input interface{}, stateRetriever stateRetriever, debug
 		debugLogger.Infof("Field", field)
 
 		if val, exists := field.Tag.Lookup("tfschema"); exists {
+			val = strings.TrimSuffix(val, ",removedInNextMajorVersion")
```

Comment:
This is fine for now, but I think we want to add a `parseStructTag` method which returns a struct to ultimately solve this - however I'm going to do that in a separate PR since that also needs to be handled in the Set too.